server/src/index.ts

```diff
@@ -80,8 +80,9 @@ app.use(
 
 const localPort = process.env.NODE_ENV === 'test' ? 5001 : 5000;
 const port = process.env.PORT || localPort;
-const server = app.listen(port, () => console.log(`Listening on port ${port}`));
-
+if (!module.parent) {
```

Comment:
what does this check do?

Reply:
@ifhuang Haha I was trying to prevent the EADDRINUSE error for tests. http://www.marcusoft.net/2015/10/eaddrinuse-when-watching-tests-with-mocha-and-supertest.html

Reply:
Add your intention of this check as a comment next to it.